.github/workflows/build-extension.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will test the build system, then build the extension and upload an artifact
+name: Test build system and upload extension
+
+on:
+  pull_request:
+    branches: '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install build dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install --no-install-recommends make gettext gnome-shell libglib2.0-bin
+
+    - name: Test extension builds from scratch
+      run: |
+        make build
+
+    - name: Build the extension bundle
+      run: |
+        make dist
+
+    - name: Test extension installs
+      run: |
+        make install
+
+    - uses: actions/upload-artifact@v3
```

Comment:
Mm, it's easy enough to build, but this saves users cloning the right fork, switching to the right branch, downloading the dependencies and then building. Building the extensions automatically guarantees that they've got the correct build of the extension, and can install it like any regular GNOME extensions, without worrying about which target will do it.

I can drop this if you'd rather, but since the extension only takes a few KB, and it's built and stored within GitHub infrastructure, I wasn't too worried about unnecessary uploads. AFAIK, it also clears old builds after enough time / rebuilds, and this limit can be configured down.